lib/src/arithmetic/values/fixed_point_value.dart

```diff
@@ -114,29 +116,17 @@ class FixedPointValue implements Comparable<FixedPointValue> {
     }
   }
 
-  /// Equal-to operation that returns a [LogicValue].
-  LogicValue eq(FixedPointValue other) =>
-      compareTo(other) == 0 ? LogicValue.one : LogicValue.zero;
+  /// Less-than operator for [FixedPointValue].
```

Comment:
What do we really want here?   I notice that LogicValue operators return a LogicValue.

But with fx and fp operators, should they return a LogicValue as well?

Reply:
by the way, do these changes not consitute a backwards-incompatible change?  that's fine, just something to note.

Reply:
I will check.  My intent was no breaking changes on this PR.

Reply:
There are breaking changes:  I changed the `FixedPointValue` comparisons to return bool instead of `LogicValue`.

Reply:
Seems like the right approach would be to NOT merge this yet, but deprecate the old `FixedPointValue` methods like `eq`, `lt`, etc. and merge that. Then do a minor release of things we have. 

THEN, we can start merging some of the breaking changes in the pipeline, including this one. which would remove those  methods and change the operators to bool instead of `LogicValue`